atlas/packages/app/src/shared/components/ToggleButton/ToggleButton.styles.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react'
+import { css } from '@emotion/core'
+
+import Button from '../Button'
+import type { ButtonStyleProps } from '../Button/Button.style'
+import { spacing, colors } from '../../theme'
+
+export type ToggleButtonStyleProps = {
+  pressedDown: boolean
+} & ButtonStyleProps
+
+const hoverTransition = ({ pressedDown, disabled = false, variant }: ToggleButtonStyleProps) =>
```

Comment:
There are some UI issues I see here.

Most importantly, Chrome will automatically add an outline to a focused element. In the following gif I'm first toggling the button and then clicking outside the button:
![togglebutton](https://user-images.githubusercontent.com/12646744/92227729-fb84fd00-eea6-11ea-8d93-d34df34d5104.gif)

As you can see, even though the button is no longer pressed down, Chrome will add an outline that makes it look very similar to the toggled state. We probably will need to improve that.

Another thing is that once the button is toggled it instantly jumps down from the hovered state (without a transition). This is a rather easy fix, it's a matter of moving the `transition` to the base style and not adding it conditionally.

However, I don't think we necessarily need to fix those right now. I think it'd be better to get this merged and open an issue describing those issues so we can tackle them later. I'll leave the decision to you